### DISCUSSION TITLE
Amalgamation File

### DISCUSSION THREAD

**Discussion by jtaylorme** [biojppm/rapidyaml#426] [General]
You speak of an amalgamation file so as to be able to use a single header file.   Apologies if I am missing the obvious but where can I find that file.   You mention it being in the package but can't find it.  Thanks.

Jim

**Comment by biojppm** [comment]
This is explained both in the main README and in the documentation: https://rapidyaml.readthedocs.io/latest/sphinx_using.html . If you're interested in the header for a particular release, just go to the release page and download the .hpp file.

**Comment by jtaylorme** [comment]
Guess I am blind or an idiot.   I can only see the ways to compile the code into a single header and not the file itself.   I will figure it out.   Can always download python if needed.  Thanks.

jim

**Comment by biojppm** [comment]
Ok, let's try again. Can you explain in clear terms a) what you are trying to accomplish, b) what you have, and c) where you are having trouble? It's hard to understand your question.

**Comment by jtaylorme** [comment]
In reading the information provided it sounded like the amalgamation file was available, already formed.   In searching the github site and the zip file of the project I couldn’t find it so thought I would ask.   I simply need the almagation file.  I am not using cmake or doing any type of “build” on the project.   I simply need the header file.   Do I have to generate that from the source or is it posted somewhere?

Thank you.

Jim

**Comment by biojppm** [comment]
> If you're interested in the header for a particular release, just go to the release page and download the .hpp file.

Eg, https://github.com/biojppm/rapidyaml/releases/tag/v0.5.0, then search for the file .hpp

**Comment by jtaylorme** [comment]
Thank you.   Finally found it.



Jim
